cmd/juju/application/diffbundle_test.go

```diff
@@ -136,13 +139,16 @@ machines:
 }
 
 func (s *diffSuite) TestIncludeAnnotations(c *gc.C) {
-	ctx, err := s.runDiffBundle(c, "--annotations", s.writeLocalBundle(c, testCharmStoreBundle))
+	ctx, err := s.runDiffBundle(c, "--annotations", s.writeLocalBundle(c, testCharmHubBundle))
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, `
 applications:
   grafana:
     missing: bundle
   prometheus:
+    charm:
+      bundle: ch:prometheus2
+      model: prometheus2
```

Comment:
This shouldn't be a diff, as they are the same thing. It could be that something is wrong in the bundle diff, already found one thing.

In the mean time, the test should be updated, so this change and the other

Reply:
I'm not sure what you mean, I had to add those lines myself to make it work, so it's a legit diff. Not sure if I'm missing    something here

Reply:
I've added a commit with the changes we discussed.